bot.js

```diff
@@ -124,14 +124,16 @@ client.on('message', msg => {
             if (msg_contains(msg,"status report")) {
                 msg.channel.send(dparse("ans_status_report", [msg.guild.memberCount]));
             }
-            if (msg_contains(msg,"list devs")) {
-                dev_ids.forEach(item => users += msg.guild.members.find(m => m.id === item) +"\n");
-                msg.channel.send(dparse("ans_list_dev", [users]));
-            }
 
-            if (msg_contains(msg,"list devs masters")) {
-                dev_master_ids.forEach(item => users += msg.guild.members.find(m => m.id === item) +"\n");
-                msg.channel.send(dparse("ans_list_dev_masters", [users]));
+            if (msg_contains(msg,"list devs")){
+                if (msg_contains(msg,"list devs masters")) {
```

Comment:
I'm sure this works, but I think a `msg.content.toLowerCase() === 'list devs';` and `msg.content.toLowerCase() === 'list devs masters';` would be more fitting. I feel these dev commands are specific enough that we shouldn't be using the contains check at all.

Reply:
No, this would not work,
because the message starts with a mention of the bot.

It will work with my implemented if structure as intended.

(In the next pull request there will be a function which analyses messages starting with a mention)

Reply:
Ah yes, I stand corrected. In that case I'll merge your pull request. 👍